README.md

```diff
@@ -14,6 +10,14 @@ The goals of code review include:
 
 See also [Atlassian's nice writeup](https://www.atlassian.com/agile/code-reviews/).
 
+## PR review checklist
```

Comment:
Is this intended to be an exhaustive list? I think it's not. Maybe add a note about that?